### ISSUE TITLE
Missing items in test set

### ISSUE THREAD

**Issue by davescroggs** [davescroggs/GrocerySalesForecasting#10]

**Comment by davescroggs** [comment]
The test set has some item_nbrs that aren't in the training set - "The test data has a small number of items that are not contained in the training data. Part of the exercise will be to predict a new item sales based on similar products."

Probably need to impute this during preprocessing, which means identifying them as missing, too.

**Comment by davescroggs** [comment]
Step_other for item numbers where not a lot of records are available. 

Create test/train set by filtering the last two months of train
